File: data/EXP01/python/2232/2232.output_2.py
# @lc app=leetcode id=2232 lang=python3
class Solution:
    def minimizeResult(self, expression: str) -> str:
        plus_index = expression.index("+")
        
        num1 = expression[:plus_index]
        num2 = expression[plus_index+1:]
        
        result = num1 + "(" + num2 + ")"
        
        return result

File: data/EXP01/golang_no_system_var/2232/2232.output_2.go
func minimizeResult(expression string) string {
    num1 := ""
    num2 := ""

    i := 0

    for expression[i] != '+' {
        num1 += string(expression[i])
        i++
    }
    i++
    for i < len(expression) {
        num2 += string(expression[i])
        i++
    }

    sum := strconv.Itoa(strconv.Atoi(num1) + strconv.Atoi(num2))

    result := ""

    if sum == "0" {
        result = "(" + num1 + "+" + num2 + ")"
    } else if len(num1) <= len(num2) {
        result = "(" + num1 + "+" + num2 + ")"
    } else {
        result = "(" + num1 + "+" + num2
        result += ")"
    }

    return result
}
